### ISSUE TITLE
Multiple Quest Rewards/ Claims in Party

### ISSUE THREAD

**Issue by xxBLoS** [FTBTeam/FTB-Sky-Odyssey#275]
* **Modpack Version**: 1.00

* **Issue**: Multiple Quest Rewards for Teammembers. My friend finished a quest and I could claim it again. So now we got 2 Transmutationtables. 

* **Is it Repeatable?**: Dont know, didnt restart our progress for now.

**Comment by dsabecky** [comment]
This is how Better Questing also worked, so not sure if this is by design or not. @xxBLoS if someone on the team isn't online when a quest is completed, are they able to turn it in as soon as they login or do they then have to recomplete the quest?

**Comment by xxBLoS** [comment]
We are online at the same time right now and each of us can claim every quest we did. I remember that the quest book should be shared. I check that.

**Comment by dsabecky** [comment]
@xxBLoS Can you try to have someone complete a quest and turn it in while someone else in the team *isn't* online, then after the turn in, have them login and see if they can turn it in, or if they have to complete the quest themselves?

**Comment by xxBLoS** [comment]
I get the quest reward. Even if I was offline while claimed. So again. We get double the rewards because we are 2 players. I think this isnt intended.

**Comment by dsabecky** [comment]
So according to ftbteam/ftb-quests#132 "team quests" are shared by everyone on the team, and as the mantra for the pack is "the pack that can't be broken", this is most likely intentional that all the quests are flagged as team quests. I'm sure that one of devs will correct me if I'm mistaken though.

TL;DR The functionality is by design of the mod, and it is *most likely* implemented this way.

**Comment by Slowpoke101** [comment]
this is working as intended.
